packages/core/src/Stores/client-store.js

```diff
@@ -371,7 +371,7 @@ export default class ClientStore extends BaseStore {
 
     @computed
     get is_authentication_needed() {
-        if (!this.account_status.status) return false;
+        if (!this.account_status?.status) return false;
```

Comment:
Why not just use the _elvis operator_ on L375 instead?

Plus this is checking for `status` not `authentication` which is needed below.

Use `return this.account_status?.authentication.needs_verification?.length;` for L375 and you can remove the check above?

Same for all the other computed values below.

Reply:
the error came at line 374 where `this.account_status` is undefined